src/scenes/Root/Header/ProfileMenu/ProfileMenu.tsx

```diff
@@ -16,6 +24,9 @@ const useStyles = makeStyles()(({ spacing }) => ({
   menuHeading: {
     padding: spacing(1, 2, 2, 2),
   },
+  listItemText: {
+    textTransform: 'capitalize',
```

Comment:
Why not just write the HTML as "Upload Manager"?
I get this idea for all caps styles, but this title case seems like normal writing..no?